packages/e2e.test.foundry-sdk-generator/src/GeneratePackageCommand.test.ts

```diff
@@ -14,18 +14,18 @@
  * limitations under the License.
  */
 
+import { __testSeamOnly_NotSemverStable__GeneratePackageCommand as GeneratePackageCommand } from "@osdk/foundry-sdk-generator";
```

Comment:
When moving the test code from a sibling to a cousin, we needed to be able to expose a test seam for GeneratePackageCommand. This name seemed scary enough for anyone that tries to use this package (no one should to be clear).